packages/react/src/primitives/shared/styleUtils.ts

```diff
@@ -240,6 +240,7 @@ const BaseStylePropsMap: Required<{ [key in keyof BaseStyleProps]: true }> = {
   transformOrigin: true,
   width: true,
   whiteSpace: true,
+  writingMode: true,
```

Comment:
Will customers really need to set this at a component level? Wouldn't they set this at the page level?